portfolio/src/main/webapp/js/background.js

```diff
@@ -0,0 +1,375 @@
+const HEIGHT_TO_WIDTH = 0.3;
+/**
+ * Runs initial set up of the background and background canvas.
+ */
+function setup() {
+  const FRAMERATE = 60;
+  const backgroundContainer = document.getElementById("background-container");
+
+  const backgroundCanvas =
+    createCanvas(backgroundContainer.clientWidth,
+      backgroundContainer.clientWidth * HEIGHT_TO_WIDTH);
+  backgroundCanvas.parent("background-container");
+
+  frameRate(FRAMERATE);
+
+  initTank();
+}
+
+let ballTank;
+/**
+ * Initializes a tank for the background.
+ */
+function initTank() {
+  const EFFECT_RAD_TO_WIDTH = 0.1;
+  const BALL_SIZE_TO_WIDTH = 0.01;
+  const NUM_BALLS = 25;
+  const INIT_BALL_VEL = 1;
```

Comment:
I'll rename it to INITIAL_BALL_VELOCITY_MAGNITUDE and add a comment in the initTank function, as it represents the magnitude for the initial velocity of all the balls. It was probably unclear because of random2D, which picks a random direction for the velocity.